core/src/components/radio/radio.scss

```
@@ -122,6 +123,11 @@ input {
   overflow: hidden;
 }
 
+:host(.in-item:not(.legacy-radio)) .label-text-wrapper {
+  margin-top: $radio-item-label-margin-top;
+  margin-bottom: $radio-item-label-margin-bottom;
```

```suggestion
  @include margin($radio-item-label-margin-top, null, $radio-item-label-margin-bottom, null);
```
Optional: Does the same thing, but now the code is consistent with other margin usage